core/che-core-api-dto/src/test/java/org/eclipse/che/dto/definitions/DTOHierarchy.java

```diff
@@ -46,5 +46,18 @@
         void setParentField(String parentField);
 
         ChildDto withParentField(String parentField);
+
```

Comment:
shadowed fields cause caused initial failure, fixed this and added them to make the behavior is consistent